azkaban-common/src/main/java/azkaban/scheduler/Schedule.java

```diff
@@ -63,7 +67,7 @@ public Schedule(int scheduleId, int projectId, String projectName,
 
     this(scheduleId, projectId, projectName, flowName, status, firstSchedTime,
```

Comment:
Thanks for review. I agree! Ray has the same concern for this. I will find a change to refactor the code in future.